ethereum-fee-prepay/src/lib.rs

```diff
@@ -46,63 +48,40 @@ pub trait EthereumFeePrepay {
     // estimate endpoints
 
     #[endpoint(payFee)]
-    fn pay_fee(
-        &self,
-        address: Address,
-        relayer: Address,
-        transaction_type: TransactionType,
-        priority: Priority,
-    ) -> SCResult<()> {
+    fn pay_fee(&self, tx_senders: Vec<Address>, relayer: Address) -> SCResult<()> {
```

Comment:
can we extend this to accept a series of whitelisted tokens ? maybe we want ETH here.

Reply:
Done on another branch.